composer.json

```diff
@@ -68,5 +68,9 @@
         "static-analysis": "psalm --shepherd --stats",
         "test": "phpunit --colors=always",
         "test-coverage": "phpunit --colors=always --coverage-clover clover.xml"
+    },
+    "conflict": {
+        "amphp/amp":"<2.6.4",
+        "symfony/console":">=7.1"
```

Comment:
I feel like a conflict is a sane path forward based on you research _(Thanks for that!)_ but I need to figure out whether we need to fix this in src to avoid the conflict or report the regression upstream

CC @laminas/technical-steering-committee

Reply:
@gsteel i found it, it's working now :)